### ISSUE TITLE
Test the mobile app which includes countly

### ISSUE THREAD

**Issue by hellais** [ooni/probe#1101]
INSTRUCTIONS:
From the countly panel:
Message -> Create Message

Select Test Users
Select Send now
Add
Message title
Message text
Either add “On click URL”
Or Select “Buttons 1" and add the url there
Tick the box and press send
Try both scenarios with app in background and app open. (4 tests in total)

Apk: https://drive.google.com/open?id=19TIi50zgo5MR52FccrEsibaPo9lWD6Cn

**Comment by hellais** [comment]
We should test what happens when the push notification backend goes offline or if we loose all the data in the mongodb.

This is to understand how much of a SPOF this is. See: https://github.com/ooni/backend/issues/397

**Comment by hellais** [comment]
I did some testing of this with my device, but was not able to receive any push notifications, while @lorenzoPrimi was able to do so.

@lorenzoPrimi will look more into this issue.

**Comment by lorenzoPrimi** [comment]
@hellais this build will work 
https://drive.google.com/open?id=1FSMBTMRslYVZCnnqbBOg1oRpjlcxpPce
Tested on two devices.

**Comment by hellais** [comment]
I managed to get this to work. Apparently there were some issues related to us having changed the app ID.

I tested the following:
- Sending a simple push notification
- Sending a push notification with an OONI Run link payload (it works and triggers the app OONI Run flow)
- Stopping the countly server and opening the app. The app works as expected without any issues

I assume that since something had to be done to reset the app ID, this is somehow quite similar to what we would have to do in the event that we were to loose all the database and have to reset it.

I would say that the only step missing is the PoC of the countly plugin which support user target country selection (https://github.com/ooni/backend/issues/378) and then we can consider this a good platform to move our mobile apps towards.
